3-enrich/scala-common-enrich/build.sbt

```diff
@@ -44,6 +44,8 @@ lazy val root = project
       Dependencies.Libraries.mysqlConnector,
       Dependencies.Libraries.jaywayJsonpath,
       Dependencies.Libraries.iabClient,
+      Dependencies.Libraries.yauaa,
+      Dependencies.Libraries.kryo,
```

Comment:
why is kryo needed? I'm afraid this is going to wreak havoc with spark

Reply:
https://github.com/nielsbasjes/yauaa/blob/master/analyzer/pom.xml#L100

Will it work in `beam-enrich` without including it?

Reply:
ah ok, weird that they would put kryo as a provided dependency